paperqa/agents/search.py

```diff
@@ -400,9 +401,7 @@ async def get_saved_object(
         return None
 
     def clean_query(self, query: str) -> str:
-        for replace in ("*", "[", "]", ":", "(", ")", "{", "}", "~", '"'):
-            query = query.replace(replace, "")
-        return query
+        return re.sub(r'[*\[\]:(){}!~^><+"\\]', "", query)
```

Comment:
I don't think we need leading/trailing `[` and `]`

Can you make a regex 101 link for this, and add to the code?

Reply:
I think if we remove the `[]` it will not work https://regex101.com/r/L1cND0/1

Reply:
I think you need to escape `*` then: `\*`